### PR TITLE
Add new quotes and media feature punctuation rules

### PR DESCRIPTION
**Pull request by jeddy3**
Adds two new rules:

1. `font-family-name-quotes` - consistent with existing quote rules.
2. `media-feature-no-missing-punctuation` - catch unmistakable errors.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Head
+
+* Removed: `stylelint < 4.3.0` compatibility.
+  * Added: `font-family-name-quotes` rule with double quotes where recommended option.
+  * Added: `media-feature-no-missing-punctuation` rule.
```
Comment (MoOx) [inline CHANGELOG.md]: why this indentation?

Comment (jeddy3) [inline CHANGELOG.md]: I did it to show that the two new rules are the reason for the removed compatibility, but now I'm wondering whether it's just confusing and unusual within a changelog :) 

I should probably remove the indentation, right?

Comment (MoOx) [inline CHANGELOG.md]: Probably, or change format of lines in the sub-list :)

**Comment by davidtheclark** [comment]
I'm good with this addition.

**Comment by davidtheclark** [comment]
What about adding no-invalid-double-slash-comments?

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint-config-standard/commit/f86159e8213d30e0922f3737e8fff1ea7c3f6a1a

**Pull request closed without merging**